brainiak/reprsimil/brsa.py

```diff
@@ -368,7 +354,7 @@ class BRSA(BaseEstimator, TransformerMixin):
         this parameter is used in a half-Cauchy prior
         on the standard deviation, or an inverse-Gamma prior
         on the variance of the GP.
-    tau2_prior: callable[[float, int, float], [float, float]],
+    tau2_prior: Callable[[float, int, float]],
```

Comment:
The return type of the `Callable` is missing:
https://docs.python.org/3/library/typing.html#typing.Callable